File: DesktopModules/ActiveForums/DAL2/WebApi/LikesDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DotNetNuke.Modules.ActiveForums
{
    public class LikesDTO
    {


        public int CommentID;
        public int UserID;
        public string LikesCount;
        public bool UserLiked;

        public string ImageName;

        public string DisLikeCount;
        public string DisLikeImageName;
    }

    public class AutoCompleteTag
    {
        public string Name { get; set; }
        //public int VocabularyID { get; set; }
    }


    public class FileStackRefernceDTO
    {
        public long FileId;
        public int ContentId;
        public string filestackurl;
        public string FileName;
    }
}

File: DesktopModules/Blog/DAL2/WebAPI/LikesDTO.vb
Public Class LikesDTO


    Public CommentID As Integer
    Public UserID As Integer
    Public LikesCount As String
    Public UserLiked As String
    Public ImageName As String
    Public UnLikeCount As String


End Class



Public Class FileStackRefernceDTO


    Public FileId As Long
    Public ContentItemID As Integer
    Public filestackurl As String
    'Public UserLiked As Boolean
    Public FileName As String
    Public UploadedBy As Integer

End Class


Public Class DTOResponse

    Public Property UserID As Integer

    Public Property ResponseText As String

    Public Property QuestionID As Integer
End Class
